sdk/textanalytics/Azure.AI.TextAnalytics/api/Azure.AI.TextAnalytics.netstandard2.0.cs

```diff
@@ -102,6 +103,11 @@ public partial class AnalyzeSentimentResultCollection : System.Collections.Objec
         public Azure.AI.TextAnalytics.TextSentiment Sentiment { get { throw null; } }
         public string Text { get { throw null; } }
     }
+    public enum Association
```

Comment:
We might need to consider to make this `EntityAssociation` an expandable enum class. Since the `Other` could be expandable possiblely.